modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -324,11 +329,11 @@ GLOBAL_VAR_INIT(successful_blood_chem, 0)
 		//20:40, 15:30, 10:20, 5:10
 		var/maturity_threshold = 20
 		if(GLOB.successful_egg_number >= GLOB.objective_egg_borer_number)
-			maturity_threshold -= 5
+			maturity_threshold -= 3
 		if(length(GLOB.willing_hosts) >= GLOB.objective_willing_hosts)
-			maturity_threshold -= 5
+			maturity_threshold -= 10
 		if(GLOB.successful_blood_chem >= GLOB.objective_blood_borer)
-			maturity_threshold -= 5
+			maturity_threshold -= 3
```

Comment:
originally, it would add up to 15 being removed-- here it removes 16; so, you may need to lessen on of the things. Maybe the 10 to a 9, or one of the 3s to a 2?

Reply:
Dropped the egg goal to a 2 since it could practically be farmed on monkeys - brings everything to 15 again.